### PR TITLE
Fix position of assert call with no message

### PR DESCRIPTION
**Pull request by nicolasstucki**
The fix is a wokaround of a bug in the inliner.

### CODE REVIEW

**Review by anatoliykmetyuk — APPROVED** [review]

**Comment by nicolasstucki** [comment]
I may have the proper fix in #10801. We can merge this one first and add #10801 after it if needed due to time constraints for M3.

**Comment by nicolasstucki** [comment]
Now that  #10801 is approved we should not merge this PR.

**Pull request closed without merging**
